### DISCUSSION TITLE
Blue rectangle when click on mobile device.

### DISCUSSION THREAD

**Discussion by mirzobedil** [tailwindlabs/tailwindcss#3023] [Help] [answered]
Help pls. When I press the Button on mobile devices, a blue rectangle appears on it. How can I remove this?

`<a href="#1" class="block px-4 py-2 rounded-full bg-transparent border border-dashed hover:bg-gray-200 hover:text-gray-500 hover:border-transparent hover:shadow-md active:bg-gray-300">Link button</a>`

![photo_2020-12-07_21-06-46](https://user-images.githubusercontent.com/45965774/101374593-2af53c80-38d0-11eb-9b59-313ed812dcd7.jpg)

![photo_2020-12-07_21-06-46 (2)](https://user-images.githubusercontent.com/45965774/101374599-2cbf0000-38d0-11eb-8c21-9780f457f142.jpg)

**Comment by mirzobedil** [comment, marked as answer]
https://github.com/tailwindlabs/discuss/issues/396

```
* { -webkit-tap-highlight-color: rgba(0,0,0,0); }
```

It works well.

**Comment by zepfietje** [comment]
I've just opened a pull request to remove the highlight by default: https://github.com/tailwindlabs/tailwindcss/pull/9050.
